includes/Modules/AdSense.php

```diff
@@ -232,7 +232,7 @@ function() use ( $client_id ) {
 	 * @return array Filtered $data.
 	 */
 	protected function amp_data_load_auto_ads_component( $data ) {
-		if ( ! $this->is_connected() ) {
+		if ( ! $this->is_connected() || ! is_singular( 'amp_story' ) ) {
```

Comment:
This must also be inversed, but furthermore it should not be here, but in `amp_content_add_auto_ads()` (line 262): exit if `! $this->context->is_amp() || is_singular( 'amp_story' )`.